Cargo.toml

```diff
@@ -14,7 +14,7 @@ members = [
 
 [dependencies.bevy]
 version = "0.8.0"
-default-features = true
+default-features = false
```

Comment:
do we need to add back any of the features here? https://github.com/bevyengine/bevy/blob/main/docs/cargo_features.md looks like we would be missing winit and render other important looking things? (good chance I'm misunderstanding)

Reply:
Ah, I'll add those to be explicit, but since our dependencies depend on those we didn't need to specify.

We should put those in there, though, I'll follow up.

Reply:
Yea it seems like it didn't cause any issues so if its unnecessary thats okay.